package.json

```diff
@@ -14,7 +15,8 @@
     "format:check": "prettier '**/*.{js,jsx,ts,tsx,html,css,scss,json,yml,md}' --check",
     "format": "prettier '**/*.{js,jsx,ts,tsx,html,css,scss,json,yml,md}' --write",
     "ng-openapi-gen": "ng-openapi-gen --input openapi.yml --output ./src/app/core/api --indexFile=true",
-    "lint": "ng lint"
+    "lint": "ng lint",
```

Comment:
what linter do we use?

Reply:
Eslint with angular plugins. I did not change anything regarding the linter in this PR. I actually have a draft pr open for the linting, since we currently do not lint the whole project.